src/classes/VOL_SharedCode.cls

```diff
@@ -31,170 +31,216 @@
+                LIMIT 999
+            ]) {
+                selectOptions.add(new SelectOption(campaignRecord.Id, campaignRecord.Name));
             }
-            return listSO;
+            return selectOptions;
         }
     }
 
     // the list of Volunteer Jobs for the specified Campaign
-    global list<SelectOption> listSOVolunteerJobsOfCampaignId(ID campaignId) {
-        list<SelectOption> listSO = new list<SelectOption>();
-        listSO.add(new SelectOption('', ''));
-        for (Volunteer_Job__c vj : [select Name, Id from Volunteer_Job__c where Campaign__c = :campaignId order by name limit 999]) {
-            listSO.add(new SelectOption(vj.id, vj.name));
+    global List<SelectOption> listSOVolunteerJobsOfCampaignId(Id campaignId) {
+        List<SelectOption> selectOptions = new List<SelectOption>{ new SelectOption('', '') };
+
+        for (Volunteer_Job__c jobRecord : [
+            SELECT Name, Id
+            FROM Volunteer_Job__c
+            WHERE Campaign__c = :campaignId
+            ORDER BY Name
+            LIMIT 999
+        ]) {
+            selectOptions.add(new SelectOption(jobRecord.Id, jobRecord.Name));
         }
-        return listSO;
+        return selectOptions;
     }
 
     // the list of Volunteer Job Shifts for the specified Job
-    global list<SelectOption> listSOVolunteerShiftsOfVolunteerJobId(ID volunteerJobId, Date dtStart, Date dtEnd, boolean fIncludeShiftName, boolean fIncludeNumberNeeded) {
-        return listSOVolunteerShiftsOfVolunteerJobIdFormat(volunteerJobId, dtStart, dtEnd, fIncludeShiftName, fIncludeNumberNeeded, null, null);
+    global List<SelectOption> listSOVolunteerShiftsOfVolunteerJobId(
+        Id jobId,
+        Date startDate,
+        Date endDate,
+        Boolean shouldIncludeShiftName,
+        Boolean shouldIncludeNumberNeeded
+    ) {
+        return listSOVolunteerShiftsOfVolunteerJobIdFormat(
+            jobId,
+            startDate,
+            endDate,
+            shouldIncludeShiftName,
+            shouldIncludeNumberNeeded,
+            null,
+            null
+        );
     }
 
     // list of select options of Shifts for the specified job, using the date & time format strings for the shifts
-    public static list<SelectOption> listSOVolunteerShiftsOfVolunteerJobIdFormat(ID volunteerJobId, Date dtStart, Date dtEnd,
-            boolean fIncludeShiftName, boolean fIncludeNumberNeeded, string strDateFormat, string strTimeFormat) {
-
-        list<Volunteer_Job__c> listVolunteerJobs = new List<Volunteer_Job__c>();
-
-        list<SelectOption> listSO = new list<SelectOption>();
-        listSO.add(new SelectOption('', ''));
-
-        // ensure valid date ranges
-        if (dtStart == null)
-            dtStart = system.today();
-        if (dtEnd == null)
-            dtEnd = system.today().addMonths(12);
-        dtEnd = dtEnd.addDays(1);
+    public static List<SelectOption> listSOVolunteerShiftsOfVolunteerJobIdFormat(

```

Comment:
This one's only public, so if you wanted to change the method name ...

Reply:
Done!

Reply:
Ran partial regression--all is working as expected!